CHANGELOG.md

```diff
@@ -1,3 +1,302 @@
+# 1.0.10-rc1 (2020-06-23)
+make histogram store data for 1 hour (#3117) (#3119)
```

Comment:
classify each change into edgeAgent, edgeHub or iotedged features/bug fixes or general like what you can find in https://github.com/Azure/azure-iotedge/releases/tag/1.0.9-rc1-2